packages/grafana-ui/src/components/SimpleSelect/SimpleSelect.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCombobox } from 'downshift';
+import React, { useState } from 'react';
+
+import { SelectableValue } from '@grafana/data';
+
+import { Input } from '../Input/Input';
+
+interface SimpleSelectProps {
```

Comment:
These could probably extend `Input`? At the very least we should pass down invalid and loading to Input.